cacerts/detect_test.go

```diff
@@ -128,5 +171,35 @@ func testDetect(t *testing.T, context spec.G, it spec.S) {
 				},
 			}))
 		})
+
+		context("BP_ENABLE_RUNTIME_CERT_BINDING is set to false", func() {
+			var result libcnb.DetectResult
+			it.Before(func() {
+				// ctx.Platform.Environment = map[string]string{"BP_ENABLE_RUNTIME_CERT_BINDING": "false"}
+				os.Setenv("BP_ENABLE_RUNTIME_CERT_BINDING", "false")
+
+				var err error
+				result, err = detect.Detect(ctx)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			it.After(func() {
+				os.Unsetenv("BP_ENABLE_RUNTIME_CERT_BINDING")
+			})
+
+			it("detect fails", func() {
+				Expect(result.Pass).To(BeFalse())
```

Comment:
I think `result.Pass` should still be true. The buildpack may still participate if a downstream buildpack explicitly requires `ca-certificates` via the build plan, as demonstrated in the test below. If no downstream buildpacks require `ca-certificates` it won't participate because provide won't have a matching require.